client/package.json

```diff
@@ -15,7 +15,7 @@
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.1",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.19.0"
+    "react-router-dom": "^6.21.1"
```

Comment:
Any reason for a dependency update or was it an accident? not a big deal right now with how small our codebase is so I'm just curious

Reply:
it was an accident.